source/user-manual/reference/tools/index.rst

```diff
@@ -45,6 +45,8 @@ Tools
 +---------------------------------------------------+----------------------------------------------------------------------------+-----------------------------+
 | :doc:`fim_migrate <fim-migrate>`                  | Migrates older FIM databases to Wazuh-DB                                   | manager                     |
 +---------------------------------------------------+----------------------------------------------------------------------------+-----------------------------+
+| :doc:`keystore-tool <keystore-tool>`              | Sets the username and password that connect Wazuh manager and indexer      | manager                     |
```

Comment:
This is the current scope, but keystore will be used with whatever sensitive information that need to be user by Wazuh server  daemons/tools

Reply:
Done in [Make requested changes](https://github.com/wazuh/wazuh-documentation/pull/7000/commits/166a6616aa43bcfd53cffd26dd40a3764a21f901) commit